README.md

```diff
@@ -462,6 +462,32 @@ export default {
 
 Will render: `My special number must equal one, and also foo`.
 
+## Raw error output
+
+By default, `ember-changeset-validations` returns the errors as plain strings.
+In some situations, it may be preferable for the developer that the library returns a description os the errors;
+internationalisation (i18n) for example, or finer-grained error output.
+
+To have `ember-changeset-validations` return such data structure, add the following to you `config/environment.js`
+
+``` 
+ENV['changeset-validations'].rawOutput = true;
```

Comment:
I am ambivalent to the specific name, but just throwing it out there that `ember-changeset-validations` could be the config name as well

Reply:
Agreed on the `rawOutput` name. Do you have a suggestion?

FWIW, I skipped the `ember-` prefix to follow other addons, such as `ember-moment` using `moment` and `ember-i18n` using `i18n`. If you still want it changed, I'm fine with it and I'll do it, just confirm, please.

Reply:
Ah good call!  I'm good with `rawOutput`.  👍